README.rst

```diff
@@ -532,8 +532,8 @@ Code example:
 .. note::
     An ``ImproperlyConfigured`` exception is raised if a menu element is already registered at the same position.
 
-    An ``ImproperlyConfigured`` exception is raised also if proper configuration is not provided based on the different types of
-    menu elements that you can register. Different types of configurations will be discussed in the next sections.
+    An ``ImproperlyConfigured`` exception is raised if proper configuration is not provided based on the different types of
```

Comment:
"if proper configuration is not provided", what about "if the supplied configuration is incorrect" or something similar?

Reply:
@nemesisdesign what do you think about this ``An ``ImproperlyConfigured`` exception is raised if supplied configuration do not match with the different types of possible configurations available( different configurations will be discussed in the next section).``

Reply:
sounds good!